frontend/app/components/input-checkbox.tsx

```diff
@@ -0,0 +1,79 @@
+import type { ReactNode } from 'react';
+
+import { InputError } from './input-error';
```

Comment:
```suggestion
import { InputError } from '~/components/input-error';
```

(for consistency with other imports)

Reply:
updated